excercsises/2_page_object/pages/home.js

```diff
@@ -9,13 +9,20 @@ var HomePage = function() {
   this.getTitle = function() {
     return browser.driver.getTitle();
   };
-  this.getItemsFromContactPage = ()=>{
-    return element(by.xpath('//*/span/h3')),element(by.xpath('//*/fieldset/h3'));
-  }
-  this.getContactText = () => {
+  this.findAdressAndgetInTouch=  async ()=>{
+    let array = [];
+    element(by.xpath('//fieldset/h3')).getText().then((function(el) {
+      array.push(el);
+  }))
+  element(by.xpath('//span[2]/h3')).getText().then((function(el) {
+      array.push(el);
+  }))
+    return array;
```

Comment:
the body code is ok, you don't need to change it. To make it cleaner I would do something like this:

this.getSectionHeaders = async () => {
	return element.all(by.xpath('//h3')).then(function(headers) {
		return headers.map(i => i.getText())
	});
}